crates/bevy_math/src/aspect_ratio/error.rs

```diff
@@ -0,0 +1,26 @@
+//! Contains error types related to aspect ratio calculations.
```

Comment:
I think stylistically this is small enough that it's going to be cleaner and simpler just to stick this at the bottom of the `aspect_ratio.rs` file, rather than splitting this apart.